ext/date/lib/timelib.h

```diff
@@ -39,22 +39,6 @@
 #include <limits.h>
 #include <inttypes.h>
 
-# ifndef HAVE_INT32_T
```

Comment:
Please provide changes to ext/date/lib/* to [upstream](https://github.com/derickr/timelib); otherwise they'll likely be overwritten by the next update.

Reply:
Will do. I didn't know this is a third-party library.

Reply:
https://github.com/derickr/timelib/pull/141